builder.go

```diff
@@ -5,75 +5,100 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"sort"
 
 	"github.com/BurntSushi/toml"
-	"github.com/pkg/errors"
 
 	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/launch"
 	"github.com/buildpacks/lifecycle/layers"
 )
 
-type Builder struct {
-	AppDir        string
-	LayersDir     string
-	PlatformDir   string
-	BuildpacksDir string
-	PlatformAPI   *api.Version
-	Env           BuildEnv
-	Group         BuildpackGroup
-	Plan          BuildPlan
-	Out, Err      io.Writer
-}
-
```

Comment:
Re-ordered some things in this file to bring struct definitions closer to their methods.